frigate/events.py

```diff
@@ -98,6 +100,8 @@ def run(self):
                         area=event_data["area"],
                         has_clip=event_data["has_clip"],
                         has_snapshot=event_data["has_snapshot"],
+                        sub_label="",
```

Comment:
I wouldn't override here. It's possible (likely) double take has set the sub_label before the event ended.